roles/openshift_master_upgrade/handlers/main.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
So that does not work either. Still hitting https://github.com/ansible/ansible/issues/20493